website/src/utils/groupFieldsByHeader.ts

```diff
@@ -11,15 +12,36 @@ const SUBMISSION_ID_FIELD: InputField = {
     required: true,
 };
 
-export const groupFieldsByHeader = (inputFields: InputField[], metadata: Metadata[]): Map<string, InputField[]> => {
+const ACCESSION_INPUT_FIELD: InputField = {
+    name: ACCESSION_FIELD,
+    displayName: 'Accession',
+    definition: 'TODO',
```

Comment:
Hmm... so we should actually make this customizable for each `name` and `accessionPrefix` (see values.yaml) - sorry for complicating this!

 But users would really need to see an example accession and we have a different `accessionPrefix` in Loculus and Pathoplexus. 
```suggestion
    definition: 'The {name, e.g. Loculus} accession (without version) of the sequence you would like to revise',
    guidance: ''
    example: '{accessionPrefix, e.g. LOC_}000P97Y`
```
You can probably leave guidance empty I think with an example and a definition this should be quite clear :-)

Reply:
I moved getGroupedInputFields into the `config.ts`, so it's easier to get these properties. Let me know what you think! https://github.com/loculus-project/loculus/pull/3478/commits/84c843ca3b9aebb6502ff94649345bf6792af67b